### ISSUE TITLE
List of radar (ids), locations, countries and coordinates

### ISSUE THREAD

**Issue by peterdesmet** [enram/timamp-etl#12]
OPERA [manages this database of radars](http://eumetnet.eu/wp-content/themes/aeron-child/observations-programme/current-activities/opera/database/OPERA_Database/index.html). The interface is a map, which makes it hard to get all the data. Is that information publicly available in table format or does someone have this?

I'm especially looking for the following fields:

- Location (Kiruna)
- Country (Sweden)
- ODIM code (sekir)
- Latitude (67.7088)
- Longitude (20.6178)

I also noticed in the sample data for Sweden, that some codes in the h5 files can deviate from the official (?) codes on the Opera website (h5 code on left / opera code on right)

* sevar = sevax?
* sehud = sehuv?
* seosu = seosd?

Why is that?

**Comment by CeciliaNilsson709** [comment]
I have all that in a table, I'll send it to you. When do you need it? I am out of office today.

Don't know why the cods differ, I also noticed that the height of the stations given there isn't _exactly_ the same as in the h5 files 🤷‍♀️. I use the codes (and heights) from the files...

**Comment by peterdesmet** [comment]
Great! Don't need it right away, sometime next week is fine. 👌

**Comment by peterdesmet** [comment]
Radar metadata added by @CeciliaNilsson709 in [radars.csv](https://github.com/enram/vp-processing/blob/master/settings/radars.csv). It contains lat/long and altitude information for the radars that are currently part of the flyway.

If more radars are added to the settings for processing, the processing will throw an error if required metadata is missing from this file.

@CeciliaNilsson709 I've removed the sunrise/sunset info from this file as it's 1) now calculated automatically and 2) not really radar metadata anyway.

Closing issue.
